content/docs/arvo/clay/marks/marks.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Overview"
+weight = 1
+template = "doc.html"
++++
+
+Clay is a typed filesystem, and we call these file types `mark`s. When talking about Hoon and Arvo we'll often talk of types like `@ud`, `(list @t)`, etc. A `mark` will specify such a type for its files, but it does more than just that - it also defines conversion routines to and from other `mark`s, as well as diff, patch, and merge routines.
```

Comment:
Somewhere on this page you should mention that marks are located in the `/mar` folder